DEVELOPER.md

```diff
@@ -55,40 +90,65 @@ git remote add upstream https://github.com/MODFLOW-USGS/modflow6.git
 
 ## Building
 
-To build MODFLOW 6 run:
+You can build modflow with Visual Studio Code tasks as described [here](.vscode/README.md).
+For the more general instructions, continue to read this section.
+
+First configure the build directory:
 
 ```shell
-# Go to the pymake directory
-cd modflow6/pymake
+# bash (linux and macOS)
+meson builddir --prefix=$(pwd) --libdir=bin
 
-# run the modflow6 pymake build script
-python makebin.py -mc ../src/ ../bin/mf6
+# cmd (windows)
+meson builddir --prefix=%CD% --libdir=bin
 ```
 
-* Results are put in the bin folder.
+For the following compilations, only this command has to be executed:
+
+```shell
+meson compile -C builddir
+```
+
+In order to run the tests the binaries have to be installed by executing:
+
+```shell
+meson install -C builddir
+```
+
+The binaries can then be found in the `bin` folder.
+
```

Comment:
MODFLOW 6 can also be built with Visual Studio using the solution and project files in the modflow6/msvs directory.

Reply:
It is possible, but running the tests with the produced binaries involves manually copying them around and the upcoming parallelization work will be disabled with it. 
Are you sure you want to mention it in the developer docs then?

Reply:
I guess the point here is that many of our potential developers use Visual Studio and it is quite good for debugging.  Since we have the solution and project files, seems worthwhile to mention it.